internal/cli/up.go

```diff
@@ -205,16 +217,25 @@ func provideWebURL(webPort model.WebPort) (model.WebURL, error) {
 	return model.WebURL(*u), nil
 }
 
-func provideSailURL() (model.SailURL, error) {
-	if !enableSail {
+func provideSailURL(mode model.SailMode) (model.SailURL, error) {
+	urlString := ""
+	switch mode {
+	case model.SailModeLocal:
+		urlString = fmt.Sprintf("//localhost:%d/", model.DefaultSailPort)
+
+	case model.SailModeProd:
+		urlString = "https://sail.tilt.dev/"
```

Comment:
hmm, will this play nice with the `Http()`/`Ws()` methods? You may have to leave off the protocol...?

Reply:
done! and added a test